File: app/src/main/java/org/wit/emergencyescape/models/LocationModel.java
package org.wit.emergencyescape.models;
import java.io.Serializable;
import java.util.HashMap;
import java.util.Map;
import java.util.UUID;

public class LocationModel implements Serializable {
    public Long id;
    public String fbId;
    public double lng;
    public double lat;
    public float zoom;
}

File: app/src/main/java/org/wit/emergencyescape/models/LocationModel.kt
package org.wit.emergencyescape.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LocationModel(var id: Long=0,
                         var fbId : String = "",
                         var lat: Double = 0.0,
                         var lng: Double = 0.0,
                         var zoom: Float = 0f) : Parcelable